check_haproxy_stats.pl

```diff
@@ -154,6 +158,7 @@ sub usage {
 my $dump;
 my $ignore_maint;
 my $ignore_drain;
+my $ignore_regex;
```

Comment:
What about a default value? I assume this flag should be optional, so if you don't set it it shouldn't affect the current behavior of the script.

Reply:
My bad.. I added check for if its unset - exactly like its done with ignore_maint and ignore_drain

Reply:
Looks good to me now. Thanks 🙂